src/main/java/net/dancier/service/MockedRecommendationsService.java

```diff
@@ -0,0 +1,30 @@
+package net.dancier.service;
+
+import net.dancier.domain.Recommendation;
+import net.dancier.domain.User;
+import net.dancier.domain.dance.Dancer;
+import net.dancier.domain.dance.School;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MockedRecommendationsService implements RecommendationsService {
```

Comment:
A short doc about mocked behaviour here would be nice.

Reply:
yes